core/primitives/src/test_utils.rs

```diff
@@ -557,13 +557,12 @@ pub fn create_test_signer(account_name: &str) -> InMemoryValidatorSigner {
 pub fn create_user_test_signer(account_name: &AccountIdRef) -> InMemorySigner {
     let account_id = account_name.to_owned();
     if account_id == near_implicit_test_account() {
```

Comment:
nit: is it better than match? (due to my lack of knowledge, I am not sure which one is preferred)

Reply:
I do not know, for me it looks less readable:
```
match account_name.to_owned() {
    account_id if account_id == near_implicit_test_account() => {
        InMemorySigner::from_secret_key(account_id, near_implicit_test_account_secret())
    }
    account_id if account_id == eth_implicit_test_account() => {
        InMemorySigner::from_secret_key(account_id, eth_implicit_test_account_secret())
    }
    account_id => InMemorySigner::from_seed(account_id, KeyType::ED25519, account_name.as_str()),
}
```
@wacban wdyt?

Reply:
Typically match would be better but here it's all dynamic values only known in runtime which complicates things. I think either is fine, I have a small preference for the `if`s.